lib/tasks/publishing_api.rake

```diff
@@ -29,6 +29,11 @@ namespace :publishing_api do
         current_base_path: '/feedback/foi',
         destination_path: '/contact/foi'
       },
+      {
+        content_id: '58b05bc2-fde5-4a0b-af73-8edc532674f8',
```

Comment:
This looks like the `content_id` for `/contact` (`/contact/foi` doesn't have it's own content item): https://www.gov.uk/api/content/contact.

It looks like there might have to be another way of putting this redirect in place.